doc/api/tls.md

```diff
@@ -971,6 +973,8 @@ added: v0.3.2
     or array of `Buffer`s of trusted certificates in PEM format. If this is
     omitted several well known "root" CAs (like VeriSign) will be used. These
     are used to authorize connections.
+  * `clientCertEngine` {string} A string with the name of an OpenSSL engine
+    which can provide the client certificate.
```

Comment:
For consistency and greppability, I'd use the same description here; it's slightly different from the other two.